vitup/src/mode/standard/controllers/main.rs

```diff
@@ -169,7 +169,13 @@ impl VitController {
             .in_settings_file(&config_file)
             .db_url(db_file.to_str().unwrap())
             .service_version(version)
-            .block0_path(self.hersir_controller.block0_file().to_str().unwrap())
+            .block0_path(Some(
```

Comment:
@dkijania Can you look here that I'm not doing something stupid?

I was just trying to fix the compiler error since `block0_path` now expects `Option<Stinig>` not `Option<&str>` as before.

Reply:
I think it's good. Another alternative is just to call `.to_string()`

```
Some(self.hersir_controller.block0_file().to_str().unwrap().to_string())
```